### ISSUE TITLE
Only one way to get to sub pages

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-brand-guidelines#219]
There is only one way to get to level 3 pages, i.e. any sub pages of the landing / section pages, for example, [Logo elements](https://govuk-brand-guidelines.netlify.app/logo-system/logo-elements/).

There is a way to get to the landing pages because the homepage has links to them. But the only method that links to the sub pages is the navigation.

There is a second way to get to them when viewing the pages on mobile. When on mobile there is a second sub navigation at the end of the main content. If this was always visible, the issue would be solved.

This fails WCAG 2.4.5 Multiple Ways. For that to pass every page (with some exceptions) needs to be found via more than one way.

It would make sense to do this together with [Sub navigations inconsistently named](https://github.com/alphagov/govuk-brand-guidelines/issues/220) as it's in the same bit of code and directly related.

## Who needs to be involved

Developer, designer, content designer

## Who needs to review

Anyone in the squad

**Comment by calvin-lau-sig7** [comment]
My reckon is we could add some sort of "In this section" table of contents on the landing pages (like the [Graphic device](https://govuk-brand-guidelines.netlify.app/graphic-device/) page).

Another option is to write some SEO-style copy that describe the different pages in each section, along with link text, but this is high effort and likely limited value.
